endpoint/endpoint.adef

```diff
@@ -11,10 +11,14 @@ processes:
     }
 }
 
+#if ${LEGATO_TARGET} = localhost
```

Comment:
Don't do that. Change the way for conditions.

Reply:
The definition files provided by legato didn't have not equal.

Reply:
Rewrite as "if ... != ... "

Reply:
?

Reply:
Cannot use not equal operator.

Reply:
Put explanation within comments.